share/spice/yacht_specs/content.handlebars

```diff
@@ -0,0 +1,32 @@
+<!-- Div for the yacht specifications -->
+<div class='thumb'>
+    
+    <div class='image'>
+        <img src='{{result_photo}}' class='profile_photo' width=300px height=200px>
+    </div>
+    <div class='info'>
+       <h5 class='yachtName'>{{name}}</h5>
+        <table>
```

Comment:
This is exactly what our `record` sub-template is designed to do!

You can remove this whole handlebars file and change the `content:` property to `content: 'record'` to use the sub-template.

Then you'll need to define a `record_keys` property inside your `data` object.

Please checkout the Bitcoin Address, Transaction and Block Spices to see how those work or checkout our documentation on the record sub-template: https://duck.co/duckduckhack/templates_reference#coderecordcode-template